packages/tools/client-debugger/client-debugger/src/FluidDebuggerLogger.ts

```diff
@@ -21,6 +30,52 @@ import { debuggerMessageSource, postMessagesToWindow, TelemetryEventMessage } fr
  * @internal
  */
 export class FluidDebuggerLogger extends TelemetryLogger {
+	/**
+	 * Accumulated data for Telemetry logs.
+	 */
+	private readonly _telemetryLog: ITelemetryBaseEvent[];
+
+	/**
+	 * Message logging options used by the debugger.
+	 */
+	private readonly messageLoggingOptions: MessageLoggingOptions = {
+		context: `FluidClientDebuggerLogger`,
+	};
+
+	/**
+	 * Handlers for inbound messages related to the debugger.
+	 */
+	private readonly inboundMessageHandlers: InboundHandlers = {
+		["GET_TELEMETRY_HISTORY"]: (untypedMessage) => {
+			this.postLogHistory();
+			return true;
+		},
+	};
+
+	/**
+	 * Event handler for messages coming from the window (globalThis).
+	 */
+	private readonly windowMessageHandler = (
+		event: MessageEvent<Partial<IDebuggerMessage>>,
+	): void => {
+		handleIncomingWindowMessage(event, this.inboundMessageHandlers, this.messageLoggingOptions);
+	};
+
+	/**
+	 * Posts a {@link IDebuggerMessage} to the window (globalThis).
```

Comment:
We should document more about the specific message means. I.e. When does this get posted? What does the data represent?